src/site/content/en/progressive-web-apps/customize-install/index.md

```diff
@@ -101,46 +124,85 @@ asking them to confirm they want to install your PWA.
 
 ```js
 buttonInstall.addEventListener('click', async () => {
-  // Hide the app provided install promotion
+  // Identify which button was clicked, this should be unique for each button.
+  installSource = 'button-header-blue';
+  // Hide the in-app install promotion.
   hideInstallPromotion();
-  // Show the install prompt
+  // Show the install prompt.
   deferredPrompt.prompt();
-  // Wait for the user to respond to the prompt
+  // Wait for the user to respond to the prompt.
   const { outcome } = await deferredPrompt.userChoice;
-  // Optionally, send analytics event with outcome of user choice
-  console.log(`User response to the install prompt: ${outcome}`);
-  // We've used the prompt, and can't use it again, throw it away
+  // Optionally, send analytics event with outcome of user choice.
+  ga('send', 'event', {
+    eventCategory: 'pwa-install',
+    eventAction: 'promo-clicked',
+    eventLabel: installSource,
+    eventValue: outcome === 'accepted' ? 1 : 0,
+  });
+  // Clear installSource if the user clicked dismiss.
+  if (outcome === 'dismissed') {
+    installSource = null;
+  }
+  // Clear deferredPrompt since it can't be used again.
   deferredPrompt = null;
 });
 ```
 
 The `userChoice` property is a promise that resolves with the user's choice.
-You can only call `prompt()` on the deferred event once. If the user
-dismisses it, you'll need to wait until the `beforeinstallprompt` event
-is fired again, typically immediately after the `userChoice` property
-has resolved.
+If the user clicks cancel, or dismiss, the install stops, the browser returns
+to step one, and the flow starts all over again. Otherwise, the browser
+installs the PWA.
+
+If you're logging the event to analytics, like before, use category
+`pwa-install`, but this time, the action should be `promo-clicked`, indicating
+the user clicked on the in-app install promotion. Use  `installSource` for the
+label to identify which install element the user clicked to start the install.
+And set the event `value` to either `1` or `0`, depending on whether the user
```

Comment:
Contrary to popular opinion, you can start a sentence with 'and'. You can also overuse it, which is why writing teachers discourage it. (It's a best practice that contributes to variety in sentence structure for goal of holding reader interest.) This is the third occurrence of this I've found and it's one of at least four. I would remove most of them. The same advice applies to 'but'.